.travis.yml

```diff
@@ -12,6 +12,18 @@ matrix:
     after_success:
       - coveralls
 
+  - language: python
```

Comment:
This looks like the correct block to me. Can you go and fix the other PR?

Reply:
Sure, or do you want to review this one and close the other? History is slightly cleaner. 

But if you have a bunch of unpublished comments I can go fix the other. 

(If you want to review this branch, I won't mess with travis anymore till after lunch)

Reply:
I just finished reviewing the other branch...

Reply:
No problem :) I'm going to wait for this to finish before changing travis on the other branch.